pages/common/pulumi-config.md

```diff
@@ -0,0 +1,28 @@
+# pulumi config
+
+> Manage configuration for a Pulumi stack.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_config/>.
+
+- View current configuration in JSON format:
+
+`pulumi config --json`
+
+- Get the value of a configuration key:
+
+`pulumi config get {{key}}`
+
+- Remove a configuration value:
+
+`pulumi config rm {{key}}`
+
+- Set value for a key from a file:
+
+`cat {{path/to/file}} | pulumi config set {{key}}`
+
+- Set secret value (e.g. API key) for a key and store/display as ciphertext:
```

Comment:
Because the example says the secret value can be an API key, reading this had me really confused for a while, because it's essentially "Set a [key] for a key". I think this sentence could use a disambiguation (i.e. *what kind of key*).
```suggestion
- Set a secret value (e.g. API key) for a configuration key and store/display as ciphertext:
```